<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManager</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>891</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件传输Demo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="FileDown">
       <attribute name="title">
        <string>文件下载</string>
       </attribute>
       <widget class="QPushButton" name="downloadBtn">
        <property name="geometry">
         <rect>
          <x>199</x>
          <y>70</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>下载文件</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>421</width>
          <height>51</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>139</x>
          <y>140</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>19</pointsize>
         </font>
        </property>
        <property name="text">
         <string>进度</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pauseBtn">
        <property name="geometry">
         <rect>
          <x>305</x>
          <y>70</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>暂停下载</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelBtn">
        <property name="geometry">
         <rect>
          <x>409</x>
          <y>70</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>取消下载</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openFolder">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>70</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>671</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="resetProgress">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>148</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="FileUpload">
       <attribute name="title">
        <string>文件上传</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>60</y>
          <width>491</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>服务器地址</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serverIpEdit_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>127.0.0.1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>服务器端口</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serverPortEdit_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>8888</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>210</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="ConnectBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>连接服务器</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disConnectbutton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>断开连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
